tests/test_wave_03.py

```diff
@@ -55,12 +55,13 @@ def test_friends_unique_movies_not_duplicated():
     # Assert
     assert len(friends_unique_movies) == 3
 
-    raise Exception("Test needs to be completed.")
-    # *************************************************************************************************
-    # ****** Add assertions here to test that the correct movies are in friends_unique_movies **********
-    # **************************************************************************************************
+    # raise Exception("Test needs to be completed.")
+    # generates dictionary of movie titles via list comprehension
+    # title is key and frequency in dictionary is value
+    unique_titles = Counter(movie["title"] for movie in friends_unique_movies)
+    assert all(value == 1 for value in unique_titles.values())
```

Comment:
Nice 👍 How could you go about writing assertions for this test without the use of the Counter object?